### PR TITLE
Fix autotuning docs

### PR DESCRIPTION
**Pull request by cli99**

### CODE REVIEW

File: deepspeed/autotuning/autotuner.py [hunk 1]
```diff
@@ -23,6 +23,9 @@
 
 
 class Autotuner:
+    """The DeepSpeed Autotuner automatically dicover the optimal DeepSpeed configuration that delievers good training speed. The Autotuner uses model information, system information, and heuristics to efficiently tune system knobs that affect compute and memory efficiencies, such as ZeRO optimization stages, micro-batch sizes, and many other ZeRO optimization configurations. It not only saves users' time but also can discover configurations better than hand-tuned method.
+    Autotuing with DeepSpeed requires no code change from DeepSpeed users. Please refer to the README for usage detils.
```
Comment (tjruwase) [inline deepspeed/autotuning/autotuner.py:27]: It not only reduces the time and resources user spend on tuning, but can

File: docs/_data/navigation.yml [hunk 2]
```diff
@@ -76,6 +78,8 @@ lnav:
         url: /tutorials/flops-profiler/
       - title: "PyTorch Profiler"
         url: /tutorials/pytorch-profiler/
+      - title: "Autotuning"
```
Comment (tjruwase) [inline docs/_data/navigation.yml:81]: We are trying to maintain the list of tutorials in the left menu in roughly alphabetically order. So Can you move this up like below?
image

File: docs/_pages/config-json.md [hunk 3]
```diff
@@ -592,11 +592,11 @@ Configuring the asynchronous I/O module for offloading parameter and optimizer s
...
     "overwrite": false,
     "metric": "throughput",
-    "num_nodes": null,
-    "num_gpus": null,
     "start_profile_step": 3,
     "end_profile_step": 5,
     "fast": false,
+    "max_train_batch_size": null,
+    "mp_size": 1,
     "num_tuning_micro_batch_sizs": 3,
```
Comment (tjruwase) [inline docs/_pages/config-json.md:600]: Is `num_tuning_micro_batch_sizs` a typo?

**Review by tjruwase — APPROVED** [review]

File: deepspeed/autotuning/autotuner.py (hunk 1 above)
Comment (cli99) [reply deepspeed/autotuning/autotuner.py:27]: Can you elaborate more on this?

Comment (tjruwase) [reply deepspeed/autotuning/autotuner.py:27]: Sure. The main suggestion is to point out that autotuner also saves user resources and not just time. 

And I suggested that as a rewrite of "It not only saves users' time but also can". But  you ignore the rewrite suggestion, but please include the resource saving benefit.

Comment (cli99) [reply deepspeed/autotuning/autotuner.py:27]: I see. Just made the change. Thanks.

**Review by jeffra — APPROVED** [review]
LGTM, other than the spellcheck issue we discussed :)

**Pull request merged**